packaging_automation/common_validations.py

```diff
@@ -1,5 +1,6 @@
 import string_utils
 from parameters_validation import parameter_validation
+import argparse
```

Comment:
There is an import in this file but nothing else.

Reply:
Fixed

Reply:
I see you moved the `import` but didn't delete it. Is it really necessary?

Reply:
Removed it actually could you check it again?

Reply:
I still see a change in this file, that doesn't look necessary. The `import re` is moved to line 1 from 3

Reply:
Rollbacked changes